events/models.py

```diff
@@ -647,7 +661,20 @@ class SuperEventType:
 
     replaced_by = models.ForeignKey('Event', on_delete=models.SET_NULL, related_name='aliases', null=True, blank=True)
 
+    maximum_attendee_capacity = models.PositiveIntegerField(
+        verbose_name=_('maximum attendee capacity'), null=True, blank=True
+    )
+    remaining_attendee_capacity = models.PositiveIntegerField(
```

Comment:
The remaining attendee capacity was left unchanged for two reasons:
- first the event organizer should be able to change it if the registration is maintained elsewhere.
- registration will most probably be the topic of upcoming sprint and then this will be discussed and possibly changed.

Reply:
Yes, I remember that I reviewed this in some pull request. Should the feature be left out if it's already known that it will be discussed in near future? I was just thinking that if any data is really collected to this field, a painful data migration is ahead. Is the event organizer going to keep this updated manually or how is the field really maintained? How the overbooking is taken care of? This `remaining_attendee_capacity` really seems something that should be automatically calculated with a signal or on post save process. Are the actual enrolments stored in LinkedEvents or are they stored in the client system? If they are in LE, it would be easy to auto calculate them - If they are in client system, it's as easy to them to update the taken seats than amount of seats left.

Reply:
agreed